docs/hosting/pantheon.md

```diff
@@ -0,0 +1,25 @@
+Pantheon
+========
+
+This project comes pre-configured for deployment on Pantheon.  
+
+Recommended Workflow
+--------------------
+We recommend that all work happens in feature branches, and goes through a Pull Request process on Github before being merged to the `master` branch.  This is our standard workflow:
+
+1. Developer starts work on a new feature by branching off of `master` into a new branch, `p-123`, where 123 is the ticket number.
```

Comment:
In order for branches to be sent to a Pantheon multidev instance, they must adhere to the following naming convention: 

- begin with `p-` (this is configurable in ../../.circleci/config.yml_
- must be all lowercase (a Pantheon limitation)
- no more than 11 characters (a Pantheon limitation)

Reply:
Yeah, I actually moved the workflow stuff into a new "workflow" docs page.  I think there will be enough there that we shouldn't bury it in the hosting section.